mockingbird/core/build.gradle.kts

```diff
@@ -1,3 +1,33 @@
 plugins {
     kotlin("jvm")
+    alias(libs.plugins.mavenPublish)
+}
+
+mavenPublishing {
```

Comment:
Just throwing out ideas: 

- You can [setup most of this in the root gradle.properties file](https://vanniktech.github.io/gradle-maven-publish-plugin/central/#configuring-the-pom) and only override the module specific stuff here
- Depending on if we'll be adding more modules that will be published we could also hide most of this in a convention plugin

Reply:
Thanks, I missed the gradle props setup.